tests/cnf/core/network/metallb/internal/frr/frr.go

```diff
@@ -230,6 +269,60 @@ func DefineBGPConfigWithStaticRouteAndNetwork(localBGPASN, remoteBGPASN int, hub
 	return bgpConfig
 }
 
+// DefineBGPConfigWithUnnumbered defines BGP config file with static route and network.
+func DefineBGPConfigWithUnnumbered(localBGPASN, remoteBGPASN int, interfaceName, peerLinkLocalAddress string,
+	advertisedIPv4Routes, advertisedIPv6Routes []string, multiHop, bfd bool) string {
+	bgpConfig := tsparams.FRRBaseConfig +
```

Comment:
I would be in favor of using go templates to produce the file because is easier way to read the content of the config.

Reply:
Again all the metallb test cases use this same configuration. But I like the idea of using a go template. I will bring this up. We are constantly doing refactoring to improve the test cases.